src/ansys/dpf/core/operators/metadata/__init__.py

```diff
@@ -4,7 +4,6 @@
 from .datasources_provider import datasources_provider 
 from .integrate_over_time_freq import integrate_over_time_freq 
 from .is_cyclic import is_cyclic 
-from .material_provider import material_provider 
```

Comment:
@rafacanton Do we know why this is removed? This seems like a retro-compatibility issue.

Reply:
@PProfizi This is the main objective of this update. This operator returns a `materials` object, which is not well exposed in pydpf-core. The objective of the work done by @MichaelNale was to make this operator private until we do that development, if ever done.